frontend/src/components/UserGenPage.js

```diff
@@ -245,6 +245,7 @@ class UserGenPage extends Component {
                     width: `${201 * fontSizeFactor}px`,
                   }}
                   tooltip={t('This is your trading avatar')}
+                  tooltipPosition='up'
```

Comment:
This one is causing the view on Android to overflow when you tap on the avatar 😄 
![photo_5929018991840442792_y](https://user-images.githubusercontent.com/111684255/195929931-3ed573d6-b5c3-4d91-8c6d-9cc0258c8025.jpg)

Reply:
Good catch :smile: . It happened to me and I did not know why!